src/components/DescriptionCocktail.jsx

```diff
@@ -0,0 +1,85 @@
+import { useState, useEffect } from 'react';
+import { Box, UnorderedList, ListItem, Spacer, Image, Stack, StackDivider, Heading, Text, Wrap, WrapItem } from '@chakra-ui/react';
+import { useParams } from 'react-router-dom';
+
+const DescriptionCocktail = () => {
+  const [cocktail, setCocktail] = useState(null);
+  const params = useParams();
+  let { drinkId } = useParams();
+  useEffect(() => {
+    if (drinkId) {
+      fetch(`https:\\www.thecocktaildb.com/api/json/v1/1/lookup.php?i=${drinkId}`)
+        .then(response => response.json())
+        .then(data => setCocktail(data.drinks[0]));
+    }
+  }, [params]);
+  if (!cocktail) return null;
+
+  return (
+    <>
+      <Wrap justify='center' spacing='50px' m={'5'}>
+        <WrapItem p={'5'} boxShadow='lg' w='500px' h='auto' borderRadius='lg' bg={'light.50'} backdropFilter='saturate(150%) blur(10px)'>
+          <Box justify='center' as='b' p={'4'} position='relative'>
+            <Stack divider={<StackDivider />}>
+              <Heading fontSize={'40px'} mt={'5'} mb={'4'} color={'light.400'}>
+                {cocktail.strDrink}
+              </Heading>
+              <Spacer />
+            </Stack>
+            <Spacer />
+            <Text fontSize='lg' color={'secondary.600'} mt={'6'} mb={'4'}>
+              <UnorderedList>
+                <ListItem>{cocktail.strAlcoholic}</ListItem>
+                <ListItem>{cocktail.strCategory}</ListItem>
+                <ListItem>{cocktail.strGlass}</ListItem>
+              </UnorderedList>
+            </Text>
+            <Text fontSize='xl' color={'light.400'} mt={'6'} mb={'3'}>
+              {' '}
+              {'Ingredients :'}{' '}
```

Comment:
```suggestion
              {'Ingredients :'}
```
Les `{' '}` vides peuvent être supprimés, ils sont créés par le `npm run format`